hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/PrecompileMintThrottlingCheck.java

```diff
@@ -67,7 +67,7 @@ public class PrecompileMintThrottlingCheck extends HapiSuite {
     private final AtomicReference<TimeUnit> unit = new AtomicReference<>(SECONDS);
     private final AtomicInteger maxOpsPerSec = new AtomicInteger(50);
     private static final int EXPECTED_MAX_MINTS_PER_SEC = 50;
-    private static final double ALLOWED_THROTTLE_NOISE_TOLERANCE = 0.05;
+    private static final double ALLOWED_THROTTLE_NOISE_TOLERANCE = 0.10;
```

Comment:
Is it related to this PR?

Reply:
Yes. It's related to this PR. @tinker-michaelj recommended updating this parameter to stabilize one of the HAPI tests.